templates/includes/sidebar.html

```diff
@@ -0,0 +1,4 @@
+<li><a href="#">Home page</a></li>
+<li><a href="#">Manufacturers</a></li>
+<li><a href="#">Cars</a></li>
```

Comment:
The link for 'Cars' is a placeholder (`#`). Replace it with the actual URL path to the cars page.